src/docs/style/logo.mdx

```diff
@@ -0,0 +1,267 @@
+
+# Design Steps
+
+## Tools
+
+We recommend vector graphics design tools such as Adobe Illustrator and Figma to design your product logo. Raster graphic tools such as Adobe Photoshop might occasionally be useful for final touches, but they are not designed for vector graphics. Do not use raster graphics tools such as Adobe Photoshop to design the foundation, because they are specialized in generating raster graphics that are not scalable.
+
+You are able to generate image assets at different resolutions to meet the need of different platforms. There are also online tools, such as [App Icon Generator](https://www.appicon.co/), to help with this process.
+
+Some platforms may have specialized tools to preview the app logo. Android, for example, encourages developers to create adaptive icons using [Android Studio's Image Asset Studio](https://developer.android.com/studio/write/image-asset-studio#create-adaptive). Amazon Alexa has a web tool,[Alexa Icon Builder](https://developer.amazon.com/en-US/blogs/alexa/alexa-skills-kit/2018/08/create-eye-catching-icons-for-your-alexa-skills-in-minutes1), to allow users to generate Alexa Skill logos quickly.
+
+## Download the App Logo Template
+
+-   Adobe Illustrator
+    -   [iOS](https://github.com/brightlayer-ui/doc-it/blob/master/src/app/assets/styles/logos/templates/iOS-IllustratorTemplate.ai?raw=true)
+    -   [Android Adaptive Icon](https://github.com/brightlayer-ui/doc-it/blob/master/src/app/assets/styles/logos/templates/Android-Adaptive-Icon-IllustratorTemplate.ai?raw=true)
+-   [PDF (for printing)](https://raw.githubusercontent.com/brightlayer-ui/doc-it/master/src/app/assets/styles/logos/App%20Logos%20Printable.pdf)
+-   [SVG](https://github.com/brightlayer-ui/doc-it/tree/master/src/app/assets/styles/logos/templates/SVG)
+-   Our [Figma Component Sticker Sheet](https://www.figma.com/community/file/1024360297793425107) also includes a component called "App Logo Template"
+
+## Glyph
+
+A glyph is a simple shape representing an object, action, or idea. Your glyph must be simple, recognizable, and scalable to ensure that your app can be easily identified on multiple devices with different resolutions.
+
+We recommend you select your base glyph from our [icon library](/style/icon-library).
+
+<ImageGrid
+    images={[Glyph1, Glyph2]}
+    captionsUnderImages={[
+        `AVOID: Avoid the use of mnemonic logos or logotypes when possible. They are often not meaningful for first-time users, and can be hard to see in small logo sizes.`,
+        `DO: Use a graphical way to represent an application or product.`,
+    ]}
+/>
+
+Do not use the Eaton logo as a glyph — your app should not be presented as the "Eaton" app. It is not unusual for a customer to have multiple Eaton products and applications installed on their devices, and if every one of those apps used the Eaton logo it becomes very difficult for a user to tell, at a glance, which application they are looking for.
```

Comment:
```suggestion
Do not use the Eaton logo as a glyph — your app should not be presented as the "Eaton" app. It is not unusual for a customer to have multiple Eaton products and applications installed on their devices — if every one of those apps used the Eaton logo it would be very difficult for a user to differentiate them.
```